Src/BootCamp.Chapter/BMI.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class BMI
+    {
+        public static void CalculateBMI(float weightInKg, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4852921)